skbio/stats/distance/_base.py

```diff
@@ -700,6 +698,7 @@ def png(self):
         """Display heatmap in IPython Notebook as PNG.
 
         """
+        from IPython.core.display import Image
```

Comment:
:+1: :+1: :+1: can we remove IPython as a dependency potentially?

Reply:
I think that is up to the maintainer. Please note that I have very little time right now to volunteer for a release, and my goal is to minimize what's being explored right now. In that light, I'd recommend punting this until the maintainer can make a decision.